docs/contribute/developer.md

```diff
@@ -3,10 +3,10 @@
 
 The following `make run` starts a [KIND](https://sigs.k8s.io/kind) cluster for you to get a local cluster for testing, see the manual install steps below for how to run against a remote cluster.
 
-> [!NOTE]
-> You will need a container runtime environment, like Docker, or experimentally, Podman, installed, to run Kind.
-> 
-> If you are on MacOS, see [Special Setup for MacOS](#special-setup-for-macos).
+!!! note
+    You will need a container runtime environment, like Docker, or experimentally, Podman, installed, to run Kind.
```

Comment:
While we're revisiting these docs, this sentence is pretty clunky with all the commas and clauses. Maybe we could change it to something like "You will need a container runtime environment like Docker to run Kind. Kind also has experimental support for Podman."